doc/openshift-guide.md

```diff
@@ -154,7 +154,7 @@ where `ec2-54-321-67-89.compute-1.amazonaws.com` is the Public Domain, and `54.3
 
 ## Tutorial Steps
 
-### Create your 3scale API Gateway using a template
+### Create your APIcast Gateway using a template
 
 1. Login into OpenShift using the `oc` command from the OpenShift Client tools you downloaded and installed in previous step. The default login credentials are _username = "admin"_ and _password = "admin"_
```

Comment:
Here is where the problem lies...
Here it is suggested to log in as admin, and later it says you should log into the console as "developer".
I followed the tutorial as "admin", and missed that "developer" part. But I'll change it to use developer throughout the tutorial.